oauth2client/django_util/storage.py

```diff
@@ -0,0 +1,34 @@
+from oauth2client.client import Storage
+from oauth2client.client import OAuth2Credentials
+
+
+def get_storage(request):
+    return DjangoSessionStorage(request)
+
+
+class DjangoSessionStorage(Storage):
+    """Storage implementation that uses Django sessions.
+    implementation.
+    """
+
+    def __init__(self, request):
```

Comment:
To make this easier when we transition to `DictionaryStorage`, make it just accept `session` instead of `request`.